ironfish/src/rpc/routes/wallet/sendTransaction.ts

```diff
@@ -100,38 +78,32 @@ router.register<typeof SendTransactionRequestSchema, SendTransactionResponse>(
       )
     }
 
-    const outputs = transaction.outputs.map((output) => {
-      let assetId = Asset.nativeId()
-      if (output.assetId) {
-        assetId = Buffer.from(output.assetId, 'hex')
-      }
-
-      return {
-        publicAddress: output.publicAddress,
-        amount: CurrencyUtils.decode(output.amount),
-        memo: output.memo,
-        assetId,
-      }
-    })
+    const outputs = request.data.outputs.map((output) => ({
+      publicAddress: output.publicAddress,
+      amount: CurrencyUtils.decode(output.amount),
+      memo: output.memo,
+      assetId: output.assetId ? Buffer.from(output.assetId, 'hex') : Asset.nativeId(),
+    }))
 
-    const fee = CurrencyUtils.decode(transaction.fee)
-    if (fee < 1n) {
-      throw new ValidationError(`Invalid transaction fee, ${transaction.fee}`)
+    const fee = CurrencyUtils.decode(request.data.fee)
+    if (fee < 0n) {
```

Comment:
I know this PR was focused on refactoring requests/response, but this also changes the behavior. I think it's fine to lump in this PR, but maybe we can add it to a bullet point of changes in the PR description for posterity.

Reply:
I'll revert this, I want to remove the minimum fee in another Pr but that has more changes so it's best not to do it here.